code/modules/cargo/exports/large_objects.dm

```diff
@@ -147,4 +148,6 @@
 	worth += C.air_contents.get_moles(GAS_MIASMA)*4 //MonkeStation Edit: Reduction of Miasma Value
 	worth += C.air_contents.get_moles(GAS_TRITIUM)*5
 	worth += C.air_contents.get_moles(GAS_PLUOXIUM)*5
+	//NUCLEIUM added. Waste Gas from RBMK Nuclear Reactor	//Monkestation Edit
+	worth += C.air_contents.get_moles(GAS_NUCLEIUM)*50
```

Comment:
50 credits/mol may be too high, but that can be considered during tests

Reply:
Reduced to 30